pyproject.toml

```
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "csm-api-client"
-version = "1.1.2"
+version = "1.1.3"
```

Bump again due to CRAYSAT-1673

```suggestion
version = "1.1.4"
```